pkgs/flatpak/com.github.rafostar.Clapper.yml

```diff
@@ -0,0 +1,23 @@
+app-id: com.github.rafostar.Clapper
+runtime: org.gnome.Platform
+runtime-version: 3.36
+sdk: org.gnome.Sdk
+command: com.github.rafostar.Clapper
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-videos
+  - --socket=pulseaudio
+  - --share=network
+
+# maybe add additinal codecs?
```

Comment:
> maybe add additinal codecs?

When you have time, please see my earlier comment about this [https://github.com/Rafostar/clapper/pull/3#issuecomment-691436334](https://github.com/Rafostar/clapper/pull/3#issuecomment-691436334). I would like to hear your opinion.

I **do not** ask you to work on them right now. I just would like to know your opinion before I merge and codecs can always be added later (the player does not work with flatpak right now, so this is not a high priority).

Reply:
That was the same Idea I had too, I just implemented it, wasn't that hard

Reply:
Thanks